src/lib/storage.js

```diff
@@ -1,5 +1,4 @@
 const ScratchStorage = require('scratch-storage');
```

Comment:
I was originally going to try to get rid of this, maybe to move toward a string "glue the parts together in the playground" model, but we'd need to restructure non-storage things too before that's feasible.